src/lcs.nim

```diff
@@ -1,20 +1,19 @@
-import os, parseopt, tables, os, strutils, utils
+import std/[parseopt, tables, strutils]
+import utils
 
 proc runMain() =
   ## main function runner
 
   var
     # variables that will be replaced to the license file
     licenseType: string
-    author: string = if getGitUsername() != "": getGitUsername() else: "" # [fullname]
+    author: string = getGitUsername() # [fullname]
```

Comment:
If `getGitUsername` fails we'll get an empty string anyway, since that's the default value of the string